run_dir/static/js/links.js

```diff
@@ -1,18 +1,18 @@
 
-function get_link_url() {
+function get_link_url(flowcell_id_reference) {
```

Comment:
This function has to be changed with the parameter in all the places its being called from which means also the lims step from the workset page and the project id from the project page. I would suggest renaming this to be more generic.

Reply:
Right, maybe just call it "reference"?